src/lib.rs

```diff
@@ -149,21 +142,12 @@ impl fmt::Display for Error {
         match self {
             &Error::InvalidPath => write!(f, "Invalid path"),
             &Error::InvalidUtf8 => write!(f, "Invalid UTF-8 string"),
+            &Error::UnsupportedOperation => write!(f, "Unsupported Operation"),
             &Error::Sys(errno) => write!(f, "{:?}: {}", errno, errno.desc()),
         }
     }
 }
 
-impl From<Error> for io::Error {
```

Comment:
Another public API removed, please list this in the `Removed` section of the CHANGELOG.